### ISSUE TITLE
focus groups, customer development, understand the audience

### ISSUE THREAD

**Issue by ryanbranciforte** [trailheadlabs/carlessca#62]
get the site in front of latinos, families, millennials and get feedback, iterate. 

site in more languages - spanish, mandarin, german, french? 

let's think more about the demographics

**Comment by jmoe** [comment]
is there funding for this?

**Comment by ryanbranciforte** [comment]
no, but there could be. my goal is to get some exposure for carless and get folks excited about it and then when there is $$ opps, we have things lined up - new destinations - SEKI, Big Sur, San Gabriel Mtns.
